.github/workflows/ci.yml

```
@@ -604,6 +604,42 @@ jobs:
           echo "$CONDA_PREFIX"
           ./uv pip install anyio
 
+  integration-test-free-threaded-linux:
+    timeout-minutes: 10
```

We could probably set a more aggressive timeout here.

```suggestion
    timeout-minutes: 5
```